jdaviz/configs/specviz/plugins/line_analysis/line_analysis.py

```diff
@@ -445,8 +446,8 @@ def _uncertainty(result):
                         uncertainty=spec_subtracted.uncertainty)
                     raw_result = analysis.line_flux(wave_spec)
                     # When flux is equivalent to Jy, lineflux result should be shown in W/m2
-                    if flux_unit.is_equivalent(u.Unit('W/m2/m'/u.sr)):
-                        final_unit = u.Unit('W/m2/sr')
+                    if flux_unit.is_equivalent(u.Unit('W/(m2 m)'/u.sr)):
+                        final_unit = u.Unit('W/(m2 sr)')
```

Comment:
does `final_unit` affect what is exposed to the user or is that logic handled elsewhere (do we need to override to show W/m2/sr in the UI)?

Reply:
This is how it will show up in the UI, but I didn't get any objections from POs. I figure get rid of the warnings this way for now, and if down the line people really hate that format I can bring back `_unit_ignore_warnings`.